src/main/webapp/resources/js/views/SearchPanel.js

```diff
@@ -117,6 +117,12 @@ Ext.define('BagDatabase.views.SearchPanel', {
             value: true,
             qtip: 'The textual description of the bag file.'
         }, {
+            boxLabel: 'Tags',
```

Comment:
Set `columns: 8` up on line 87 so the addition of an extra checkbox doesn't cause the line to wrap.